ddl/ddl_worker.go

```diff
@@ -977,14 +983,15 @@ func (w *worker) runDDLJob(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64,
 	failpoint.Inject("mockPanicInRunDDLJob", func(val failpoint.Value) {})
 
 	if job.Type != model.ActionMultiSchemaChange {
-		logutil.Logger(w.logCtx).Info("[ddl] run DDL job", zap.String("job", job.String()))
+		logutil.Logger(w.logCtx).Debug("[ddl] run DDL job", zap.String("job", job.String()))
```

Comment:
Please don't modify this level. This is a log that prints for each state, and if you delete it, it's hard to debug later.

Reply:
Kind of noise since it will print every time we come here.